### ISSUE TITLE
[BUG] allowCrossUidActivitySwitchFromBelow no resource idientifier issue on 2.10.0

### ISSUE THREAD

**Issue by mamiiblt** [iBotPeaches/Apktool#3753]
### Information
1. **Apktool Version (`apktool -version`)** - 2.10.0
2. **Operating System (Mac, Linux, Windows)** - Linux (Manjaro)
3. **APK From? (Playstore, ROM, Other)** - Playstore
4. **Java Version (`java --version`)** - openjdk "21.0.5" 2024-10-15 LTS

### Stacktrace/Logcat

```
Instafel Patcher 4.2 (alpha)

---------------------------------------------------------

   t_code: generate_instafel
   t_name: Generate Instagram APKs
   t_desc: Generate & sign apk files
   t_auth: mamiiblt

---------------------------------------------------------

2024-12-25 19-32-59 IFL : Loaded 4 step succesfully.

2024-12-25 19-32-59 IFL : Step 1 : Remove drawables.bin
2024-12-25 19-32-59 IFL : drawables.bin file succesfully deleted.
2024-12-25 19-32-59 IFL : Step 1 success

2024-12-25 19-32-59 IFL : Step 2 : Build Apk
2024-12-25 19-32-59 LOG : I: Using Apktool 2.10.0 with 4 thread(s).
2024-12-25 19-32-59 LOG : I: Checking whether sources has changed...
2024-12-25 19-32-59 LOG : I: Checking whether sources has changed...
2024-12-25 19-32-59 LOG : I: Checking whether sources has changed...
2024-12-25 19-32-59 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether resources has changed...
2024-12-25 19-33-00 LOG : I: Building resources...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-00 LOG : I: Checking whether sources has changed...
2024-12-25 19-33-11 LOG : W: /workspaces/instafel_generator/workdir/ig_decompiled/AndroidManifest.xml:95: error: No resource identifier found for attribute 'allowCrossUidActivitySwitchFromBelow' in package 'android'
W:
2024-12-25 19-33-11 LOG : brut.androlib.exceptions.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/tmp/brut_util_Jar_61206357731400169497555696337727644061.tmp, p, --forced-package-id, 127, --min-sdk-version, 28, --target-sdk-version, 35, --version-code, 376212237, --version-name, 362.0.0.0.146, --no-version-vectors, -F, /tmp/APKTOOL5046630640725381575.tmp, -e, /tmp/APKTOOL354025447996375460.tmp, -0, arsc, -I, /home/codespace/.local/share/apktool/framework/1.apk, -S, /workspaces/instafel_generator/workdir/ig_decompiled/res, -M, /workspaces/instafel_generator/workdir/ig_decompiled/AndroidManifest.xml]
2024-12-25 19-33-11 IFL : Error while generating apk
2024-12-25 19-33-11 IFL : Step 2 failed

---------------------------------------------------------
```

### Steps to Reproduce
1. `apktool  d -b instagram.apk -o ig_decompiled`
2. `instafel modifications......`
3. `apktool b --use-aapt1 instagram.apk -o ig_build.apk`

### Frameworks
If this APK is from an OEM ROM (Samsung, HTC, LG). Please attach framework files
(`.apks` that live in `/system/framework` or `/system/priv-app`)

### APK
If this APK can be freely shared, please upload/attach a link to it.

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? yeah
4. If you are trying to install a modified apk, did you resign it? yeyeh
5. Are you using the latest apktool version? yep

### Extra

Idk this is happens but you mentioned this issue in [here](https://github.com/iBotPeaches/Apktool/issues/3716), but `apktool empty-framework-dir` doesn't work for me too.

**Comment by mamiiblt** [comment]
actually i think it's not same issues mentioned in [here](https://github.com/iBotPeaches/Apktool/issues/3716)

**Comment by iBotPeaches** [comment]
I see this asset in the internal framework being released in Apktool 2.11.0 - so I believe this is already resolved.

```
    resource 0x010106a1 attr/allowCrossUidActivitySwitchFromBelow PUBLIC STAGED_ID=0x01bd0019
```

**Comment by mamiiblt** [comment]
> I see this asset in the internal framework being released in Apktool 2.11.0 - so I believe this is already resolved.
> 
> ```
>     resource 0x010106a1 attr/allowCrossUidActivitySwitchFromBelow PUBLIC STAGED_ID=0x01bd0019
> ```

yeah but in 2.10.0 it's occurs too, you can check it im sure about this is an bug

also i recreated frameworks too
